docker-compose.yml

```diff
@@ -11,4 +11,19 @@ services:
     ports:
       - ${DB_PORT}:5432
     volumes:
-      - ./docker-data/db:/var/lib/postgresql/data
+      - pgdata:/var/lib/postgresql/data
+
+  app:
+    build: .
+    depends_on:
+      - db
+    restart: always
+    ports:
+      - 5173:5173
+
+    environment:
+      AUTH_SECRET: ${AUTH_SECRET}
```

Comment:
All of the environment variables will need to be exposed. See `.env.example` for a list of them.

Reply:
> All of the environment variables will need to be exposed. See `.env.example` for a list of them.

I am not really sure I understand what you mean by exposed ?!